bin/test/i18n/test_i18n_script_utils.rb

```diff
@@ -120,6 +103,47 @@ def test_to_js_locale_returns_formated_js_locale
     end
   end
 
+  describe '.log_error' do
+    let(:log_error) {described_class.log_error(error_class, error_message)}
+
+    let(:error_class) {'expected_error_class'}
+    let(:error_message) {'expected_error_message'}
+
+    it 'returns error class and message' do
+      described_class.expects(:puts).with('[expected_error_class] expected_error_message').once
+      _(log_error)
+    end
+  end
+
+  describe '.fix_yml_file' do
+    let(:fix_yml_file) {described_class.fix_yml_file(provided_yaml_file_path)}
+
+    let(:provided_yaml_file_path) {'provided_yaml_file_path'}
+    let(:provided_yaml_file_data) {"---\nen-US:\n  data\n"}
+    let(:expected_yaml_output) {%Q["en-US":\n  data\n]}
+
+    before do
+      FileUtils.mkdir_p File.dirname(provided_yaml_file_path)
+      File.write provided_yaml_file_path, provided_yaml_file_data
+    end
+
+    it 'yml file should be reformatted correctly' do
+      _(fix_yml_file)
+      expect(File.read(provided_yaml_file_path)).must_equal expected_yaml_output
```

Comment:
I think we lost a stub on the read call here. we should do what the original tests do and just set up call expectations for read and write methods (so we don't actually write a file) and just call the method after setting those up.